app/src/main/java/io/intrepid/contest/screens/contestcreation/categorieslist/CategoryViewHolder.java

```diff
@@ -41,6 +41,12 @@ void bindCategory(Category category) {
     }
 
     @Optional
+    @OnClick(R.id.delete_category_icon)
+    void onCategoryDeleteClicked() {
+        // TODO: 2/15/17
```

Comment:
The date in the todo isnt particularly useful, we want to have an actual statement about what it should do.